src/app/components/Item/RequestButtons.jsx

```diff
@@ -17,11 +16,9 @@ const RequestButtons = ({item, bibId, searchKeywords, appConfig, page}) => {
   const isAeon = item.aeonUrl && features.includes('aeon-links')
 
   const getItemRecord = (e) => {
-    let gaLabel = 'Item Holding';
     if (page === 'SearchResults') gaLabel = 'Search Results';
     if (page === 'BibPage') gaLabel = 'Item Details';
     if (page === 'SubjectHeadingShowPage') gaLabel = 'Subject Heading Details';
```

Comment:
Leaving these three lines makes this function empty with no operation. It seems safe to remove the entire function, but take a closer look at line 33 where it's called: `onClick={ifAvailableHandler(e => getItemRecord(e, bibId, item.id), item.available)}` I'm not sure what `ifAvailableHandler` does so audit it and make sure it's still needed or not.